events/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.db.models.signals import pre_save
+
+
+class Event(models.Model):
+
+    class Colors(models.TextChoices):
+        RED = "#FF0000", "Red"
+        PINK = "#FFC0CB", "Pink"
+        ORANGE = "#FFA500", "Orange"
+        YELLOW = "#FFFF00", "Yellow"
+        PURPLE = "#800080", "Purple"
+        GREEN = "#008000", "Green"
+        BLUE = "#0000FF", "Blue"
+        BROWN = "#A52A2A", "Brown"
+        WHITE = "#FFFFFF", "White"
+        GRAY = "#808080", "Gray"
+        BLACK = "#000000", "Black"
+
+    title = models.CharField(max_length=50)
+    location = models.CharField(blank=True, max_length=50)
+    description = models.TextField(blank=True)
+    date_time_start = models.DateTimeField()
+    date_time_end = models.DateTimeField()
+    color = models.CharField(
+        max_length=7,
+        choices=Colors.choices,
+        default=Colors.BLACK,
+    )
+
+    def __str__(self) -> str:
+        return f"{self.title} - {self.date_time_start}"
+
+
+def validate_blank_title(title):
+    if title is None:
```

Comment:
```suggestion
    if not title:
```
if the "title" is equal to an empty string the validation will pass

as you can see:
In [3]: a = ""

In [4]: if a is None:
   ...:     print("a is None")
   ...: else:
   ...:     print("a is NOT None")
   ...: 
a is NOT None

Reply:
done, changed it as suggested and now validation raises error